config/scootconfig/workers_modules.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/scootdev/scoot/workerapi/client"
 )
 
+type ClientTimeout time.Duration
+
```

Comment:
would prefer to have a DefaultClientTimeout = time.Minute here to make it easily configurable across the board